### DISCUSSION TITLE
Possibly add Chaotic-AUR support (?)

### DISCUSSION THREAD

**Discussion by itsHanibee** [prasanthrangan/hyprdots#870] [Ideas]
It'll give us a pretty impressive install/update time reduction considering we don't have to build packages locally, Just something to consider.

**Comment by kRHYME7** [comment]
@itsHanibee you're right! Installation becomes faster. BTW Hyprdots do have the installation in ```  /usr/lib/hyprdots-ctl/Install_ChaoticAUR  ```  But this will only popout if someone is installing using ``` Hyprdots-ctl ```

**Reply by itsHanibee** [reply]
ah gotcha, we could try and make it run on first install and subsequent runs of the install script can just pull the updates from chaotic!
the suggestion is because i just tried out chaotic-aur a few days ago and _almost all_ my packages moved over to binaries from chaotic with an update and there are no obvious regressions i can notice. (all the hyprdots packages moved over, i installed an obscure kernel binary from aur, thats the only one that didn't move over)

i was looking at a previous discussion that you guys had with DarkXero and he was suggesting the same thing among some other neat changes :P

**Reply by kRHYME7** [reply]
Only issue with this are in terms of security. But using the AUR and chaotic exhibits the same issue.


>  i installed an obscure kernel binary from aur, thats the only one that didn't move over)

cases like this, looks like no one uploaded a chaotic AUR binary on there data base.
